services/shuttle-poem/Cargo.toml

```diff
@@ -9,8 +9,14 @@ keywords = ["shuttle-service", "poem"]
 [workspace]
 
 [dependencies]
-poem = { version = "1.3.55" }
+poem-1 = { package = "poem", version = "1.3.55", optional = true }
+poem-2 = { package = "poem", version = "2.0.0", optional = true }
 shuttle-runtime = { path = "../../runtime", version = "0.36.0", default-features = false }
 
 [dev-dependencies]
 tokio = { version = "1.26.0", features = ["macros", "rt-multi-thread"] }
+
+[features]
+default = ["poem-1"]
```

Comment:
This default can be changed to poem 2 when shuttle-poem 0.37 is released. It's a breaking change so it shouldn't happen before then.

The README should be updated at the same time.